src/Bot.ts

```diff
@@ -32,45 +54,62 @@ export default class Bot extends CommandoClient {
                 prefix: false,
                 ping: false,
             })
-            // .registerGroups([
-            //     ['admin']
-            // ])
-            // .registerCommandsIn(path.join(__dirname, './commands'))
-    }
+            .registerGroups([['pingable', 'Commands for Pingable users']]);
+        // .registerCommandsIn(path.join(__dirname, './commands'))
 
-    public async start() {
-        await this.login(CONFIG.bot.token)
+        // TODO replace this with .registerCommandsIn()
+        //      for some reason I can't get the commands to load from there
+        //      so I am forced to register them manually
+        this.registry.registerCommands([ExtendTimeout, ClearTimeout]);
```

Comment:
```js
  .registerGroups([
    ['public'],
    ['staff'],
    ['logs'],
    ['events'],
  ])
  .registerDefaults()
  .registerCommandsIn(path.join(__dirname, 'commands'));
  ```
  
  Example

Reply:
No longer needed, since this is changing to sapphire. **but** I did actually try that, and for some reason the bot wouldn't load the commands. I'm pretty sure I was setting up groups or something wrong, or something stupid though.